src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -786,6 +786,7 @@ const mongoObjectToParseObject = (className, mongoObject, schema) => {
       case '_email_verify_token_expires_at':
       case '_account_lockout_expires_at':
       case '_failed_login_count':
+      case '_old_passwords':
```

Comment:
@bhaskaryasa 
As you are storing password history in this column name, I think `_password_history` might be a better name for this variable instead of `_old_passwords`. It also flows with the entire feature.
WDYT?